src/pages/[platform]/build-a-backend/add-aws-services/interactions/chatbot/index.mdx

```diff
@@ -0,0 +1,68 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Interact with bots',
+  description: 'Learn more about how to integrate chat bot interactions into your application using Amplify.',
+  platforms: [
+    'javascript',
+    'react-native',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+  ],
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+
+### Send messages to bot
+You can send a text message to chatbot backend with `send()` command. The method returns a promise that includes the chatbot response.
+
+```javascript
+import { Interactions } from '@aws-amplify/interactions';
+
+let userInput = "I want to reserve a hotel for tonight";
+
+(async () => {
+  // Provide a bot name and user input
+  const response = await Interactions.send({
+    botName: "TheBotName",
+    message: userInput
+  });
+
+  // Log chatbot response
+  console.log(response.message);
+})()
```

Comment:
```suggestion
const userInput = "I want to reserve a hotel for tonight";

// Provide a bot name and user input
const response = await Interactions.send({
  botName: "TheBotName",
  message: userInput
});

// Log chatbot response
console.log(response.message);
```

let -> const since it's not reassigned later
removed IIFE in favor of top-level await in the browser context